.github/workflows/stale.yml

```diff
@@ -0,0 +1,28 @@
+name: Stale
+
+on:
+  issues:
+    types: [reopened]
+  schedule:
+    - cron: '*/60 * * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    env:
+      ACTIONS_STEP_DEBUG: true
+    steps:
+      - uses: actions/stale@v3.0.17
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+          stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+          stale-issue-label: 'stale'
+          stale-pr-label: 'stale'
+          days-before-stale: 10
+          days-before-close: 1
```

Comment:
Those limits seem to me a bit short?
@oznu any opinion?

Reply:
just copied ui settings